### ISSUE TITLE
Apollo locked automatically

### ISSUE THREAD

**Issue by HiiamIE** [christianselig/apollo-bugs#1443]
**Describe the bug**
Apollo will be locked automatically if i play videos. It only happened if i turned on lock with face id & password

**To Reproduce**
1. Go to 'face id & passcode'
2. Click on 
3. Scroll down to 'post and tap video with sound'

**iOS version**: IOS 15
**Apollo version**: 1.11
**iOS device**: iPhone XR 
**Jailbroken**: not jailbreak 
**Reproducible:** every time 

**Expected behavior**
It should no locked automatic

**Screenshots**
If possible, add screenshots to help show the problem.
![18900AF9-D768-41D3-880D-CF21F8D6FAD1](https://user-images.githubusercontent.com/91165515/134263458-28fe56ef-204f-4ca5-95c4-0e5f8b4b0229.png)
![61BF6BE1-3E8E-4E95-B9F2-AF244E609D22](https://user-images.githubusercontent.com/91165515/134263467-3d9763fe-fe19-4bac-9c40-2ca629a64da4.png)
![07FA6048-F5ED-4C6A-8B6D-82A0EBA3F119](https://user-images.githubusercontent.com/91165515/134263476-8be9ef21-7aaf-419a-9d95-155b2ba79e29.png)

**Video**
If possible, attach a video to help show the problemw. You can attach a video by uploading it to Streamable, Gfycat, YouTube, etc. and pasting the URL here.

**Additional context**

**Comment by HiiamIE** [comment]
Yes! That happened to me too

**Comment by tgansertdoe** [comment]
Same w fingerprint ID on ios15 iPad Pro 10.5. Can’t open any links, really.

**Comment by acalla4** [comment]
Came to report this issue too

**Comment by bengillam** [comment]
Getting same here too iPhone 12 Pro iOS 15 

only way I can get back in is to swap to another app then back to Apollo and it will show unlock screen again Home Screen and back doesn’t seem to be enough

this is on release build, TestFlight still an older build by looks of it

**Comment by Rendu1** [comment]
> Getting same here too iPhone 12 Pro iOS 15
> 
> only way I can get back in is to swap to another app then back to Apollo and it will show unlock screen again Home Screen and back doesn’t seem to be enough
> 
> this is on release build, TestFlight still an older build by looks of it

Same thing with me. I have the same exact issue

**Comment by Pablo-Merino** [comment]
Yup, I have the same issue. I believe it happens when I open a video and tap the enable sound button. I've attached a video that shows this behavior.

I've realized this also happens when opening some URL on the internal browser as well.

Playing videos:
https://user-images.githubusercontent.com/482747/134347608-682383b5-902b-431b-a31a-38fae171d162.MOV

Opening links in internal browser:
https://user-images.githubusercontent.com/482747/134348053-1f027a42-ef12-4365-a832-6d019803a84b.MOV

Thank you!

**Comment by justinan7** [comment]
> **Describe the bug**
> Apollo will be locked automatically if i play videos. It only happened if i turned on lock with face id & password
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> 
> 1. Go to 'face id & passcode'
> 2. Click on
> 3. Scroll down to 'post and tap video with sound'
> 4. See error
> 
> **iOS version**: IOS 15
> **Apollo version**: 1.11
> **iOS device**: iPhone XR
> **Jailbroken**: not jailbreak
> **Reproducible:** every time
> 
> **Expected behavior**
> It should no locked automatic
> 
> **Screenshots**
> If possible, add screenshots to help show the problem.
> ![18900AF9-D768-41D3-880D-CF21F8D6FAD1](https://user-images.githubusercontent.com/91165515/134263458-28fe56ef-204f-4ca5-95c4-0e5f8b4b0229.png)
> ![61BF6BE1-3E8E-4E95-B9F2-AF244E609D22](https://user-images.githubusercontent.com/91165515/134263467-3d9763fe-fe19-4bac-9c40-2ca629a64da4.png)
> ![07FA6048-F5ED-4C6A-8B6D-82A0EBA3F119](https://user-images.githubusercontent.com/91165515/134263476-8be9ef21-7aaf-419a-9d95-155b2ba79e29.png)
> 
> **Video**
> If possible, attach a video to help show the problemw. You can attach a video by uploading it to Streamable, Gfycat, YouTube, etc. and pasting the URL here.
> 
> **Additional context**

Same here. Also experienced with opening links. iPhone 12. 15.0

**Comment by Nanorithm** [comment]
This is also happening to me as well.

IPhone 12 pro max, ios 15.0

**Comment by MrClean157** [comment]
Also having this, though it happens in several other instances, too.

If trying to reply to a thread, lock comes up. Ask videos, gifs, and enabling video sound. 

For video, one work around is to swipe up so it’s playing picture-in-picture, then expand to full screen. Another workaround I found to fix the other instances of this is to leave the app, then reopen it and it’ll authenticate and remove the lock overlay.

**Comment by mrcustard** [comment]
The same thing happens if you attempt to search Reddit or search a subreddit. 

iPhone 12 Pro Max iOS 15

**Comment by inf0l** [comment]
> Yup, I have the same issue. I believe it happens when I open a video and tap the enable sound button. I've attached a video that shows this behavior.
> 
> I've realized this also happens when opening some URL on the internal browser as well.
> 
> Playing videos:
> https://user-images.githubusercontent.com/482747/134347608-682383b5-902b-431b-a31a-38fae171d162.MOV
> 
> Opening links in internal browser:
> https://user-images.githubusercontent.com/482747/134348053-1f027a42-ef12-4365-a832-6d019803a84b.MOV
> 
> Thank you!

Happens to me as well. When I click to enable sound on a video, I get the lock screen. Swiping up results in video playing PiP on Home Screen which I can then maximize again. 

iPhone 11
iOS 15.0
